src/main/java/org/isf/admission/rest/AdmissionController.java

```diff
@@ -191,7 +190,7 @@ public ResponseEntity<AdmissionDTO> getCurrentAdmission(@RequestParam("patientCo
 	 */
 	@GetMapping(value = "/admissions/allAdmittedPatients", produces = MediaType.APPLICATION_JSON_VALUE)
 	public ResponseEntity<List<AdmittedPatientDTO>> allAdmittedPatients()
-			throws OHServiceException {
+					throws OHServiceException {
 		LOGGER.info("Get all admitted patients");
 
```

Comment:
We could keep just the "/admissions/admittedPatients" endpoint, then make query parameters optionels and return all admitted patients in case any parametter is specified.

Reply:
"Admitted patients" is different from "Admissions", the change is specifically needed to separate the two concepts as stated in the OHCS-92: "_The “ui” should then just process (count) a list of AdmissionDTO (not AdmittedPatientsDTO)_"

Reply:
OK !